tools/spark-reader-v1/src/test/java/com/vesoft/nebula/NebulaReaderExample.java

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula;
+
+import com.facebook.thrift.protocol.TCompactProtocol;
+import com.vesoft.nebula.common.Type;
+import org.apache.spark.SparkConf;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NebulaReaderExample {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NebulaReaderExample.class);
+
+    public static void main(String[] args) {
+        SparkConf sparkConf = new SparkConf();
+        sparkConf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
+        sparkConf.registerKryoClasses(new Class[]{TCompactProtocol.class});
+        SparkSession sparkSession = SparkSession
+                .builder()
+                .config(sparkConf)
+                .master("local[4]")
+                .getOrCreate();
+
+        Dataset<Row> vertexDataset = sparkSession
+                .read()
+                .format("com.vesoft.nebula.reader.NebulaDataSource")
+                .option("importType", Type.VERTEX.getType())
+                .option("ip", "127.0.0.1")
+                .option("spaceName", "test")
+                .option("storagePort", "45500")
+                .option("returnCols", "course=name;building=name;student=name")
```

Comment:
So ... you want to return multi properties from multi schemas right?

Reply:
ye, the result is like this: 

![edgeschema](http://ww1.sinaimg.cn/large/006avC6gly1gi5az3wjq7j306b0akdfx.jpg)

if the edge or vertex doesn't have the properties, the default value is null。

Reply:
Actually I'm not sure collect multi schema's column into one DataFrame is a good choice.

Reply:
it can collect single schema' column into one DataFrame by set returnCols like person=name,age.

Reply:
I know. But collect different column from multi schema make me confuse.

Reply:
yep, i will think about how to solve this result, then should i close thi pr?